src/components/VVideoPlayer/index.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <video
+    @[mouseoverName]="turnOn"
+    @[mouseleaveName]="tyrnOff"
+    @pause="pause"
+    @ended="pause"
+    @keyup="changeSpeed"
+    ref="video"
+    id="video"
+    :poster="previewImageLink"
+    :controls="!isPreview"
+    :[muted]="isPreview"
+    title="Buttons W and S can change speed of your player"
+  >
```

Comment:
I like your code style. Everything from new line, I'm trying to do the same

Reply:
Thank you) I just use my prettier and code formatter for that : )